tests/verkle/eip4762_verkle_gas_witness/test_codecopy_generic.py

```diff
@@ -202,15 +202,15 @@ def _generic_codecopy(
     codecopy_code = Op.CODECOPY(0, offset, size) * repeat
     pre = {
         TestAddress: Account(balance=1000000000000000000000),
-        TestAddress2: Account(code=codecopy_code + Op.PUSH0 * (code_size - len(codecopy_code))),
+        TestAddress2: Account(code=codecopy_code + Op.PUSH0 * max(0, size - len(codecopy_code))),
```

Comment:
`code_size` could be smaller than `len(codecopy_code)`. (e.g: `code_size=0`)
The idea of `code_size` is that `TestAddress2` code has at a minimum `code_size` bytes.